src/WireMock.Net.Abstractions/BuilderExtensions/RequestModelBuilder.cs

```diff
@@ -0,0 +1,68 @@
+// ReSharper disable once CheckNamespace
+namespace WireMock.Admin.Mappings;
+
+/// <summary>
+/// RequestModelBuilder
+/// </summary>
+public partial class RequestModelBuilder
```

Comment:
Hi, 
on the `IRequestBulder` the `WithPath` is typed `string[]` which makes it very clear that there can be multiple paths set that'll match. Atm it is `object?`.

Some others:
`WithStatusCode` not using `HttpStatusCode`
`WithDelay` => `Timespan`
`WithBodyxxx` => using all `object` notation on the `IResponsebuilder` sometimes having more parameters.

`WithBodyAsJson(object body, Encoding? encoding = null, bool? indented = null)`